docs/shipping/App360/App360.md

```diff
@@ -122,26 +126,18 @@ processors:
   batch:
   tail_sampling:
     policies:
-      [
-        {
-          name: policy-errors,
-          type: status_code,
-          status_code: {status_codes: [ERROR]}
-        },
-        {
-          name: policy-slow,
-          type: latency,
-          latency: {threshold_ms: 1000}
-        }, 
-        {
-          name: policy-random-ok,
-          type: probabilistic,
-          probabilistic: {sampling_percentage: 10}
-        }        
-      ]
+      - name: policy-errors
+        type: status_code
+        status_code: {status_codes: [ERROR]}
+      - name: policy-slow
+        type: latency
+        latency: {threshold_ms: 1000}
+      - name: policy-random-ok
+        type: probabilistic
+        probabilistic: {sampling_percentage: 10}
   metricstransform/metrics-rename:
     transforms:
-      - include: ^duration.*$$
+      - include: ^duration.*$
```

Comment:
The `$$` is ok because it's the [syntax otel asks to use](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/metricstransformprocessor/README.md#group-metrics)